core/src/main/scala/scalapb/zio_grpc/RequestContext.scala

```diff
@@ -3,26 +3,30 @@ package scalapb.zio_grpc
 import io.grpc.MethodDescriptor
 import io.grpc.Attributes
 import io.grpc.ServerCall
+import zio.stm.TSemaphore
 
 final case class RequestContext(
     metadata: SafeMetadata,
     responseMetadata: SafeMetadata,
     authority: Option[String],
     methodDescriptor: MethodDescriptor[_, _],
-    attributes: Attributes
+    attributes: Attributes,
+    canSend: TSemaphore
```

Comment:
This looks like an implementation detail of the backpressure that leaks into the public interface that is visible to users. What can be done to hide it?

Reply:
Yes 🤦 Thanks for pointing out, fixed in the next commit!